test-data/unit/check-expressions.test

```diff
@@ -1122,8 +1122,8 @@ f = lambda: ''.x
 f = lambda: ''
 [out]
 main:3: error: "str" has no attribute "x"
-main:4: error: Incompatible return value type (got "str", expected "int")
 main:4: error: Incompatible types in assignment (expression has type Callable[[], str], variable has type Callable[[], int])
+main:4: error: Incompatible return value type (got "str", expected "int")
```

Comment:
the churn here (and in some other *.test files) is because now errors on earlier columns come first